python/requirements_compiled.txt

```diff
@@ -0,0 +1,549 @@
+array-record==0.4.0
+arrow==1.2.3
+asttokens==2.2.1
+astunparse==1.6.3
+async-exit-stack==1.0.1
+async-generator==1.10
+async-timeout==4.0.2
+asyncmock==0.4.2
+attrs==21.4.0
+autograd==1.6.2
+autorom==0.6.1 ; platform_machine != "arm64"
+autorom-accept-rom-license==0.6.1
+aws-sam-translator==1.71.0
+aws-xray-sdk==2.12.0
+ax-platform==0.3.2 ; python_version >= "3.8"
+azure-cli-core==2.40.0
+azure-cli-telemetry==1.0.8
+azure-common==1.1.28
+azure-core==1.27.1
+azure-identity==1.10.0
+azure-mgmt-compute==23.1.0
+azure-mgmt-core==1.4.0
+azure-mgmt-network==19.0.0
+azure-mgmt-resource==20.0.0
+babel==2.12.1
+backcall==0.2.0
+backoff==1.10.0
+base58==2.0.1
+bayesian-optimization==1.4.3
+bcrypt==4.0.1
+beautifulsoup4==4.11.1
+black==22.10.0
+bleach==6.0.0
+blessed==1.20.0
+bokeh==2.4.3
+boto==2.49.0
+boto3==1.26.76 ; python_version >= "3.8"
+botocore==1.29.76
+botorch==0.8.5
+cached-property==1.5.2
+cachetools==5.3.1
+catboost==1.2
+certifi==2023.5.7
+cffi==1.15.1
+cfn-lint==0.77.10
+charset-normalizer==3.2.0
+chess==1.7.0
+chex==0.1.7
+click==8.1.4
+cloudpickle==2.2.0
+cma==3.2.2
+cmaes==0.9.1
+colorama==0.4.6
+coloredlogs==15.0.1
+colorful==0.5.5
+colorlog==6.7.0
+comet-ml==3.31.9
+comm==0.1.3
+commonmark==0.9.1
+configobj==5.0.8
+configspace==0.7.1
+contextlib2==21.6.0
+contourpy==1.1.0
+crc32c==2.3
+crcmod==1.7
+cryptography==38.0.1
+cycler==0.11.0
+cython==0.29.32
+dask==2022.10.1 ; python_version >= "3.8"
+databricks-cli==0.17.7
+datasets==2.0.0
+debugpy==1.6.7
+decorator==5.1.1
+deepspeed==0.8.3
+defusedxml==0.7.1
+deprecated==1.2.14
+dill==0.3.6
+distlib==0.3.6
+distributed==2022.10.1
+dm-tree==0.1.8
+dnspython==2.3.0
+docker==6.1.3
+docker-pycreds==0.4.0
+docutils==0.17.1
+dopamine-rl==4.0.5
+dragonfly-opt==0.1.7
+dulwich==0.21.5
+ecdsa==0.18.0
+entrypoints==0.4
+et-xmlfile==1.1.0
+etils==1.3.0
+evaluate==0.4.0
+everett==3.2.0
+exceptiongroup==1.1.2
+execnet==2.0.0
+executing==1.2.0
+fairscale==0.4.6
+fastapi==0.99.1
+fasteners==0.18
+fastjsonschema==2.17.1
+feather-format==0.4.1
+ffmpy==0.3.0
+filelock==3.12.2
+flake8==3.9.1
+flake8-bugbear==21.9.2
+flake8-comprehensions==3.10.1
+flake8-quotes==2.0.0
+flaml==1.1.1
+flask==2.1.3
+flask-cors==4.0.0
+flatbuffers==2.0.7
+flax==0.6.11
+fonttools==4.40.0
+fqdn==1.5.1
+freezegun==1.1.0
+frozenlist==1.3.3
+fsspec==2023.5.0
+future==0.18.3
+gast==0.4.0
+gcs-oauth2-boto-plugin==3.0
+gin-config==0.5.0
+gitdb==4.0.10
+gitpython==3.1.31
+glfw==2.6.2
+google-api-core==2.11.1
+google-api-python-client==2.65.0
+google-apitools==0.5.32
+google-auth==2.21.0
+google-auth-httplib2==0.1.0
+google-auth-oauthlib==0.4.6
+google-cloud-core==2.3.3
+google-cloud-storage==2.5.0
+google-crc32c==1.5.0
+google-pasta==0.2.0
+google-reauth==0.1.1
+google-resumable-media==2.5.0
+googleapis-common-protos==1.59.1
+gpustat==1.1
+gpy==1.10.0
+gpytorch==1.10
+gradio==3.11.0 ; platform_system != "Windows"
+grapheme==0.6.0
+graphql-core==3.2.3
+graphviz==0.20.1
+greenlet==2.0.2
+grpcio==1.50.0 ; python_version < "3.10"
```

Comment:
@pcmoritz this is still below 1.53.0

Reply:
@krfricke is it possible to get grpcio `>=1.53.0` ?

Reply:
Yes, will update. This is still based on master from yesterday before 229c70bfdd22683db5430d6568b116d62ffd79e1